gateway/1.4/modules/ROOT/pages/_partials/task-config-liveness-probe.adoc

```diff
@@ -0,0 +1,99 @@
+//tag::intro1[]
+
+Flex Gateway offers the following CLI command to test the health of a Flex Gateway:
+
+[source,ssh]
+----
+flexctl probe --check=liveness
+----
+
+This command returns either a status code of `0` for a healthy Flex Gateway or `1` for an unhealthy Flex Gateway.
```

Comment:
```suggestion
If the Flex Gateway instance is healthy, the command returns a status code of `0`. Otherwise it returns a `1`.
```
Shortening sentences.